xml/libvirt_managing.xml

```diff
@@ -979,12 +979,18 @@ Metadata:       yes
    </para>
    <important>
     <para>
-     For the post-copy type of live migration, &kvm; hosts with kernels version
-     5.11 or newer require setting the
+     In order to use the post-copy live migration option, &kvm; hosts with
+     kernel version 5.11 or newer require setting the
      <varname>unprivileged_userfaultfd</varname> system value to
-     <literal>1</literal> before the migration starts:
+     <literal>1</literal>:
     </para>
 <screen>&prompt.sudo;sysctl -w vm.unprivileged_userfaultfd=1</screen>
+    <para>
+     Since kernel versions prior to 5.11 did not require setting
+     unprivileged_userfaultfd to use the post-copy option, libvirt provides
+     the setting in <filename>/usr/lib/sysctl.d/60-qemu-postcopy-migration.conf</filename>
+     sysctl file to preserve the old behavior.
```

Comment:
Not obvious what 'sysctl' files are and maybe not needed, it's just a file :-)
```suggestion
     file to preserve the old behavior.
```

Reply:
Same here